### PR TITLE
Added filter to separate speed bumps

### PR DESCRIPTION
**Pull request by noppanit**
Adding another constraint to separate speed bumps from each other. The default is 1 paragraph.

### CODE REVIEW

**Comment by noppanit** [comment]
@goldenapples #reviewmerge

File: inc/class-speed-bumps.php [hunk 1]
```diff
@@ -90,6 +91,7 @@ public function register_speed_bump( $id, $args = array() ) {
 		add_filter( $filter_id, '\Speed_Bumps\Constraints\Text\Minimum_Text::content_is_long_enough_to_insert', 10, 4 );
 		add_filter( $filter_id, '\Speed_Bumps\Constraints\Content\Injection::this_speed_bump_not_already_inserted', 10, 4 );
 		add_filter( $filter_id, '\Speed_Bumps\Constraints\Content\Injection::no_speed_bump_inserted_here', 10, 4 );
+		add_filter( $filter_id, '\Speed_Bumps\Constraints\Content\Injection::paragraph_not_far_away_enough', 10, 4 );
```
Comment (goldenapples) [inline inc/class-speed-bumps.php]: It'd be `paragraph_far_enough_away`. The constraint name describes the condition that needs to be met in order to insert.

**Comment by goldenapples** [comment]
Looks good to me. I'll fix the one nit I had about method naming and merge.

**Comment by noppanit** [comment]
@goldenapples cool changed it! Thanks Than.

**Comment by goldenapples** [comment]
Thanks! merging now.

**Comment by danielbachhuber** [comment]
@noppanit This is getting really cool :+1:

**Comment by noppanit** [comment]
Thanks @goldenapples. :-)

**Pull request merged**